angular.json

```diff
@@ -46,7 +46,7 @@
               {
                 "glob": "**/*",
                 "input": "./shared/images/",
-                "output": "/assets/images/"
+                "output": "/assets/"
```

Comment:
This should not go like this.

Assets folder is common folder for all different types of assets like Images, Audios, Icons, Styles etc..

We should not add images directly to the assets folder. It should go as assets/images only.
Later any other assets like audios comes then it will go into assets/audios & it continues for other types.